ECHO-COVID19.ipynb

```diff
@@ -7,12 +7,31 @@
     "# Tracking EPA's COVID-19 non-enforcement policy\n",
     "In late March 2020, EPA released a memo announcing that it would not penalize regulated industries that fail to meet their monitoring and reporting requirements due to COVID-19. Specifically EPA has said that it:\n",
     "\n",
-    "\"is not seeking penalties for noncompliance only in circumstances that involve routine monitoring and reporting requirements, if, on a case-by-case basis, EPA agrees that such noncompliance was caused by the COVID-19 pandemic.\"\n",
+    "> \"is not seeking penalties for noncompliance only in circumstances that involve routine monitoring and reporting requirements, if, on a case-by-case basis, EPA agrees that such noncompliance was caused by the COVID-19 pandemic.\"\n",
     "\n",
-    "This may have a number of public and environmental health impacts. See our response to EPA's memo here: https://envirodatagov.org/epas-covid-19-leniency-is-a-free-pass-to-pollute/\n",
+    "This may have a number of public and environmental health impacts if facilities respond by increasing their emissions and discharges. See our response to EPA's memo here: https://envirodatagov.org/epas-covid-19-leniency-is-a-free-pass-to-pollute/\n",
     "\n",
-    "What is the effect of EPA's non-enforcement of Clean Air Act and Clean Water Act permits? Using this notebook, you can track how facilities' monitoring and reporting of air and water hazards has changed in your state over the past few months, compared to previous years. \n",
+    "What is the effect of EPA's non-enforcement of Clean Air Act and Clean Water Act permits? Using this notebook, you can track how facilities' releases - as well as monitoring and reporting - of air and water hazards has changed in your state over the past few months, compared to previous years. \n",
     "\n",
+    "Essentially, there are three scenarios we may see playing out:\n",
+    "\n",
+    "_Monitoring and reporting violations_\n",
+    "- 1. facilities that do not report (we can track this)....but do still meet their permit limits (yet we can't know this specifically, precisely because they didn't report)\n",
+    "- 2. facilities that do *not* report (we can track this)....and actually exceed their limits (yet we can't know this specifically, precisely because they didn't report)\n",
```

Comment:
```suggestion
    "- 2. Facilities that do *not* report (we can track this)....and actually exceed their limits (yet we can't know this specifically, precisely because they didn't report)\n",
```